packages/sdks-tests/src/e2e-tests/custom-components.spec.ts

```diff
@@ -24,7 +24,7 @@ test.describe('Custom components', () => {
     packageName,
     sdk,
   }) => {
-    test.skip(!['angular', 'react'].includes(sdk));
+    test.skip(!['angular-16', 'react'].includes(sdk));
```

Comment:
This looks like a wrong check: you're comparing a `packageName` (`angular-16`, etc.) to `sdk` (should always be `angular` for all 3 angular e2e servers)

Reply:
This is the case for many other checks changed in `.spec.ts` files in this PR.

Reply:
ahh my bad, I might have used replace all. Thanks for the catch, have updated it